test/dynamics/backend/test_dynamics_backend.py

```diff
@@ -550,6 +552,40 @@ def test_metadata_transfer(self):
         self.assertDictEqual(res.results[1].header.metadata, {"key1": "value1"})
 
 
+    def test_valid_measurement_properties(self):
+        """Test that DynamicsBackend instantiation always carries measurement instructions"""
+
+        # Case where no measurement instruction is added manually (which is the case for self.simple_backend)
+        instruction_schedule_map = self.backend_2q.target.instruction_schedule_map()
+        assert instruction_schedule_map.has(instruction='measure', qubits=self.simple_backend.num_qubits)
```

Comment:
What exactly is this line checking? I'm confused by setting `qubits=self.simple_backend.num_qubits`; what happens if you pass `qubits=x` for `x` an index outside of the qubit index range?

Reply:
I see this line has been removed.